seednsort.md

```diff
@@ -1,28 +1,20 @@
 ## Seeding and Sorting Overview
 
-## What do Seeders/Sorters do?
 Seeders and Sorters canvass the resources of a given government agency, identifying important URLs. They identify whether those URLs can be crawled by the Internet Archive's webcrawler. If the URLs are crawlable, the Seeders/Sorters nominate them to the End-of-Term (EOT) project, otherwise they add them to the Uncrawlable spreadsheet using the project's Chrome Extension.
 
 ## Choosing the website
 The Seeders/Sorters team will use the EDGI subprimer systems ([found here](https://envirodatagov.org/agency-forecasts/)), or a similar set of resources, to identify important/at risk data. Talk to the DataRescue organizers to learn more.
 
 ## Canvassing the website and evaluating content
-- Start exploring the website assigned, identifying important URLs.
+- Start exploring the website assigned, identifying important URLs. 
 - Decide whether the data on a page or website subsection can be [automatically captured by the Internet Archive webcrawler](./what-heritrix-does.md).
-- The best source of information about the seeding and sorting process is represented at [https://envirodatagov.org/](https://envirodatagov.org/), see:
-- [Understanding What the Internet Archive Webcrawler Does](https://docs.google.com/document/d/1PeWefW2toThs-Pbw0CMv2us7wxQI0gRrP1LGuwMp_UQ/edit)
-- [Seeding the Internet Archive’s Webcrawler](https://docs.google.com/document/d/1qpuNCmBmu4KcsS_hE2srewcCiP4f9P5cCyDfHmsSAVU/edit))
+- The best source of information about the seeding and sorting process is represented at [https://envirodatagov.org/](https://envirodatagov.org/).
+- See also [here](https://docs.google.com/document/d/1PeWefW2toThs-Pbw0CMv2us7wxQI0gRrP1LGuwMp_UQ/edit)
```

Comment:
From a readability perspective I don't think you want to obscure document titles under "here", could we revert this section of changes?